Tasks/JavaToolInstallerV0/task.json

```diff
@@ -148,6 +149,14 @@
             "argumentFormat": ""
         }
     },
+    "restrictions": {
+        "commands": {
+            "mode": "restricted"
+        },
+        "settableVariables": {
+            "allowed": ["JAVA_HOME_*"]
```

Comment:
JavaToolInstaller also calls toolLib.prependPath, which updates PATH variables - so I guess we also need to put it here.